cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help

```diff
@@ -76,6 +76,9 @@ OPTIONS
        --debug
            Run tests in remote debugging mode.
 
+       --with-choreo
+           Package Choreo observability extension in the executable jars created while building.
```

Comment:
```suggestion
           Package the Choreo observability extension in the executable JAR files created while building.
```

Reply:
Fixed in 77fd6022bdbaf2870da71ae578be26ded14c77b5